packages/remix/package.json

```diff
@@ -18,9 +18,18 @@
   "types": "dist/index.d.ts",
   "sideEffects": false,
   "exports": {
-    ".": "./dist/index.js",
-    "./ssr.server": "./dist/ssr/index.js",
-    "./api.server": "./dist/api/index.js"
+    ".": {
```

Comment:
@anagstef @LekoArts 
Could you please verify that this is correct?

Reply:
LGTM. Looking at https://webpack.js.org/guides/package-exports/#conditional-syntax

> The last condition in the object might be the special "default" condition, which is always matched.

I'm not 100% sure if we could use `require` here, but `default` seems like a safe bet for now

Reply:
Exactly - used default as it should mimic the `exports` behavior we had before. Thank you!